stress-job.groovy

```diff
@@ -5,7 +5,7 @@ pipelineJob("Stress-job") {
         stringParam('BRANCH', '', 'James branch')
         stringParam('SIMULATION', '', 'Gatling simulation')
         stringParam('GITHUB_SIMULATIONS', '', 'Github repository containing gatling simulations')
-        stringParam('PROFILE', 'reference', 'Blobs store target: reference (default, for over-time performance tracking), swift or s3')
+        stringParam('PROFILE', 's3', 'Blobs store target: s3 (default) or reference (for over-time performance tracking, but disabled for now)')
```

Comment:
Why do we need this profile ever again?

Reply:
Hmmm my guess would be when we rebuild the corpus with s3, to be able to choose between `reference` (so with corpus` and `s3` (without corpus) ? Could probably think of a better renaming though :)